### DISCUSSION TITLE
Error: 'publish' is not a registered subcommand of 'site'

### DISCUSSION THREAD

**Discussion by rclilly** [EasyEngine/easyengine#1535] [Support Requests] [answered]
I was trying to make my site available over the Internet as per [Share your local sites on internet with site publish](https://easyengine.io/handbook/share-your-local-sites-on-internet/) but when I try i get:

`Error: 'publish' is not a registered subcommand of 'site'. See 'ee help site' for available subcommands.`

When I do `ee help site` 'publish' is not among the options. Is this option still avaiable?

For reference, I am running `EE 4.2.0` on Ubuntu 20.04.

**Comment by mrrobot47** [comment, marked as answer]
Hey, that command has now been updated to `ee site share` instead of `ee site publish`.
Please try `ee site share <site-name>` and it should work.

Thanks for pointing out the issue with the article, it has now been updated with the new command.

**Reply by rclilly** [reply]
Thank you!
